src/ape/cli/arguments.py

```diff
@@ -19,10 +19,22 @@ def _require_non_existing_alias(value):
 
 
 def existing_alias_argument(account_type: Optional[Type[AccountAPI]] = None):
+    """
+    A ``click.argument`` for an existing account alias.
+
+    Args:
+        account_type (type[:class:`~ape.api.accounts.AccountAPI`], optional):
+          If given, limits the type of account the user may choose from.
```

Comment:
What do you think of this?
```suggestion
          If given, limits the account plugin the user may choose accounts from.
```

Reply:
I wanted to incorporate plugins in there somehow, because that is the common use-case.  However, a plugin could have multiple account API implementations and this method can actually wall-off to just one of those too if it wanted.